src/fullnode.rs

```diff
@@ -37,11 +40,41 @@ pub enum OutFile {
     Path(String),
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+/// Fullnode configuration to be stored in file
+pub struct Config {
+    pub network: ReplicatedData,
```

Comment:
Can you name that field whatever you're planning to rename`ReplicatedData` to? Maybe `NodeInfo`?